micro-rdk-server/esp32/main.rs

```diff
@@ -39,44 +40,42 @@ mod esp32 {
     }
 
     pub(crate) fn main_esp32() {
-        micro_rdk::esp32::esp_idf_svc::sys::link_patches();
+        esp_idf_svc::sys::link_patches();
+        esp_idf_svc::log::EspLogger::initialize_default();
 
-        micro_rdk::esp32::esp_idf_svc::log::EspLogger::initialize_default();
+        esp_idf_svc::sys::esp!(unsafe {
+            esp_idf_svc::sys::esp_vfs_eventfd_register(
+                &esp_idf_svc::sys::esp_vfs_eventfd_config_t { max_fds: 5 },
+            )
+        })
+        .unwrap();
+
+        #[cfg(feature = "qemu")]
+        let _network = {
```

Comment:
I'm not really sure how this works exactly since we're not passing it into some form of `serve_web_with_external`

Reply:
Let's not worry too much about qemu right now.